sdcm/sla/sla_tests.py

```diff
@@ -0,0 +1,152 @@
+#!/usr/bin/env python
+import logging
+import time
+import traceback
+
+from sdcm.sct_events import Severity
+from sdcm.sct_events.workload_prioritization import WorkloadPrioritizationTestEvent
+from sdcm.sla.libs.sla_utils import SlaUtils
+from sdcm.utils.loader_utils import DEFAULT_USER, DEFAULT_USER_PASSWORD
+from test_lib.sla import create_sla_auth
+
+LOGGER = logging.getLogger(__name__)
+
+
+class Steps(SlaUtils):
+    # pylint: disable=too-many-arguments
+    def run_stress_and_validate_scheduler_runtime_during_load(self, tester, read_cmds, prometheus_stats, read_users,
+                                                              stress_queue, sleep=600):
+        # pylint: disable=not-context-manager
+        with WorkloadPrioritizationTestEvent(
```

Comment:
as for `WorkloadPrioritizationTestEvent` sound like it's a general step event, not something specific to SLA (and we could use for other things)

Reply:
`TestStepEvent` ok?

Reply:
sounds o.k. to me, and can be helpful in other places

Reply:
renamed